### PR TITLE
fix broken links, remove wikilinks

### PR DESCRIPTION
**Pull request by LWinterberg**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -2,13 +2,12 @@ There are several ways to contribute to Audacity:
 
 ## Developing
 
-Audacity is mostly coded in C++. You'll need to be at least somewhat comfortable in it to start contributing code. 
+Audacity is mostly coded in C++. You'll need to be at least somewhat comfortable in it to start contributing code.
 
-If you want to develop Audacity: 
+If you want to develop Audacity:
```
Comment (CBID2) [inline CONTRIBUTING.md:7]: [Nitpick] I think adding the verb “help” would facilitate a more collaborative working environment for this project.
```suggestion
If you want to help develop Audacity:
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -20,21 +19,21 @@ Audacity supports a wide variety of plug-in APIs, namely [Nyquist](https://manua
...
 
-When reporting bugs, try to find the most general form of it. For example, if you encounter a bug when amplifying a clip 2 hours into a project, try to see if it also happens when you're using a different effect (Normalize, for example), and if it also happens if the clip is near the beginning. 
+When reporting bugs, try to find the most general form of it. For example, if you encounter a bug when amplifying a clip 2 hours into a project, try to see if it also happens when you're using a different effect (Normalize, for example), and if it also happens if the clip is near the beginning.
 
-Bugs must be reproducible. If you can't find steps to reproduce a bug, try asking if you went wrong somewhere [in the Audacity forum](https://forum.audacityteam.org/index.php) instead. 
+Bugs must be reproducible. If you can't find steps to reproduce a bug, try asking if you went wrong somewhere [in the Audacity forum](https://forum.audacityteam.org/) instead.
```
Comment (CBID2) [inline CONTRIBUTING.md:24]: [Proofreading]
```suggestion
Bugs **must** be reproducible. If you can't find steps to reproduce one, try posting about your experience [in the Audacity forum](https://forum.audacityteam.org/) instead. 
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -20,21 +19,21 @@ Audacity supports a wide variety of plug-in APIs, namely [Nyquist](https://manua
...
+Bugs must be reproducible. If you can't find steps to reproduce a bug, try asking if you went wrong somewhere [in the Audacity forum](https://forum.audacityteam.org/) instead. 
 
 ## Translating
 
 See [our translators page](https://www.audacityteam.org/community/translators/) for more information on how to translate Audacity. 
 
 ## Feedback & Feature requests
 
-If you have anything you thing Audacity can do better, you can voice it in [the relevant section in the forum](https://forum.audacityteam.org/viewforum.php?f=25). If you have a very concrete idea something in the code which should be added or changed, you can also  [make an enhancement request in the issue tracker](https://github.com/audacity/audacity/issues/new/choose).
+If you have anything you thing Audacity can do better, you can voice it in [the relevant section in the forum](https://forum.audacityteam.org/c/feedback-and-discussion-forum/adding-features-to-audacity/22). If you have a very concrete idea something in the code which should be added or changed, you can also  [make an enhancement request in the issue tracker](https://github.com/audacity/audacity/issues/new/choose).
```
Comment (CBID2) [inline CONTRIBUTING.md:32]: [Proofreading]
```suggestion
Got some ideas on how to improve Audacity? Share them at [the relevant section in the forum](https://forum.audacityteam.org/c/feedback-and-discussion-forum/adding-features-to-audacity/22). If you have a very concrete idea like something in the code which should be added or changed, you can also  [make an enhancement request in the issue tracker](https://github.com/audacity/audacity/issues/new/choose).
```

File: INSTALL [hunk 4]
```diff
@@ -3,5 +3,4 @@ Compilation and install instructions
 
 Compilation instruction for Audacity is available at https://github.com/audacity/audacity/blob/master/BUILDING.md
 
-You can ask for help with compilation problems at:
-   http://forum.audacityteam.org/viewforum.php?f=19 .
+You can ask for help with compilation problems at: https://forum.audacityteam.org/c/programming-and-development/compiling-audacity/18
```
Comment (CBID2) [inline INSTALL:6]:
```suggestion
You can ask for help with compilation problems in the [Programing and Development section of our forum](https://forum.audacityteam.org/c/programming-and-development/compiling-audacity/18)
```

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -2,13 +2,12 @@ There are several ways to contribute to Audacity:
 
 ## Developing
 
-Audacity is mostly coded in C++. You'll need to be at least somewhat comfortable in it to start contributing code. 
+Audacity is mostly coded in C++. You'll need to be at least somewhat comfortable in it to start contributing code.
 
-If you want to develop Audacity: 
+If you want to develop Audacity:
 
 * Build instructions can be found in the BUILDING.md
```
Comment (CBID2) [inline CONTRIBUTING.md:9]: [Typo]
```suggestion
* Building instructions can be found in the BUILDING.md
```

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -2,13 +2,12 @@ There are several ways to contribute to Audacity:
 
 ## Developing
 
-Audacity is mostly coded in C++. You'll need to be at least somewhat comfortable in it to start contributing code. 
+Audacity is mostly coded in C++. You'll need to be at least somewhat comfortable in it to start contributing code.
```
Comment (CBID2) [inline CONTRIBUTING.md:5]: [Proofreading]
```suggestion
Audacity is mostly coded in C++. You'll need to be at least somewhat comfortable with using this programming language to start contributing code.
```

**Comment by Ryan-Miller-Audacity** [comment]
Reviewing links within the App and they appear to be working as expected and point to the correct locations.

Was this question mark tooltip ever clickable? Or is it just meant to be a hovered tooltip? It looks the same as the other help links seen throughout the preferences pages, but looking back all the way to 3.3.0 (when it was introduced) it is not clickable. 

image

Besides that one note things seems to operate as expected.

**Comment by Ryan-Miller-Audacity** [comment]
Also took a few minutes to review some of the updated links in contributing.md and readme.md as well and the links there look good.

**Comment by Ryan-Miller-Audacity** [comment]
> Reviewing links within the App and they appear to be working as expected and point to the correct locations.
> 
> Was this question mark tooltip ever clickable? Or is it just meant to be a hovered tooltip? It looks the same as the other help links seen throughout the preferences pages, but looking back all the way to 3.3.0 (when it was introduced) it is not clickable.
> 
> image
> 
> Besides that one note things seems to operate as expected.

I went ahead and approved this since I'm pretty certain when we implemented this that it was the intention to be a hover only tooltip. If I misremember we can retrack and fix.

**Pull request merged**